man/setkey.Rd

```diff
@@ -139,5 +143,18 @@ setindex(DT, A)
 setindex(DT, B)
 indices(DT)           # get indices single vector
 indices(DT, vectors = TRUE) # get indices list
+
+# Example illustrating the importance of using the dot (`.`) syntax with integer keys:
+DT <- data.table(id = 2:1)
+
+# Incorrect Usage:
+setkey(DT, id)
+subset_value <- 1
+DT[subset_value]  # Incorrect: treats `subset_value` as an row number
+
+# Correct Usage:
+# To correctly subset based on the key column `id`, use the dot (`.`) syntax:
+DT[.(subset_value)]  # Correct: explicitly matches `subset_value` against `id`
```

Comment:
I'm not sure about the word "explicit" here
I think `DT[.(subset_value), on="id"]` would be explicit and current code (without on arg) could be considered "implicit" can you please change here and above? or just delete explicit?

Reply:
done :)